test/fixtures/search-suggestions-sass.json

```diff
@@ -58,10 +58,7 @@
                     },
                     "script_score": {
                         "lang": "groovy",
-                        "script": "doc[\"package.name.raw\"].value.equals(text) ? 100000 + _score : _score",
```

Comment:
I think we will loose the boost on the exact matches

Reply:
the reasoning for this was that I believe this type of query already requires that the prefix matches? the boost on `package.name.raw` was resulting in packages like `requ` and `asyn` bubbling up above `async` and `request`.

Perhaps a compromise would be making _boost exact match_ configurable? my gut though is that just ranking on score might be good enough.

Reply:
I think for suggestions, exact matches are less important.  If you start typing `requ` you probably want to see `request` sooner.

Reply:
I think that not doing the boost will have some undesired effects. For instance, if someone searches for `request` the first recommendation should be `request`, despite a module named `request-handler` has best score. Having this behind an option would let us toggle the behavior and allow us to experiment with it. I will probably enabled this PR lands so that I keep the current behavior.